### PR TITLE
CP-41450: Prepare branch for sample code move

### PR DESCRIPTION
**Pull request by kc284**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+This repository contains sample code for interacting with XenServer using the
+XenServer SDK.
+
+## Available languages
+
+There are examples for each of the five programming or scripting languages of
+the Xenserver SDK:
+
+- [C](c)
```
Comment (danilo-delbusso) [inline README.md:11]: maybe we should add link that redirect to the SDKs once we have it all figured out. i.e. when/if we publish them. Python already has this as it's built together with xapi and so the package is available in the releases section of the repository

Comment (kc284) [reply README.md:11]: Definitely. This is the initial commit, we'll need to expand it in future. The READMEs per language as well will need better instructions on how to get the binaries etc,

**Review by katshann — APPROVED** [review]

**Review by katshann — APPROVED** [review]

**Review by katshann — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+* christopher.lancaste1@cloud.com danilo.delbusso@cloud.com konstantina.chremmou@cloud.com
```
Comment (danilo-delbusso) [inline .github/CODEOWNERS:1]: I'll put in a comment so we don't forget:

Waiting for Chris' email to be updated

**Review by danilo-delbusso — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (kc284) [reply .github/CODEOWNERS:1]: Need input from @CitrixChris on this.

**Review by danilo-delbusso — APPROVED** [review]

**Pull request merged**